tests/groupTest.js

```diff
@@ -88,7 +89,6 @@ describe('Main: Currently testing group management,', function () {
       await oc.login()
       return oc.groups.getGroups().then(data => {
         expect(typeof (data)).toBe('object')
-        expect(data.indexOf('admin')).toBeGreaterThan(-1)
```

Comment:
It does not feel good for me to remove this check
maybe we can add the "admin" group in ocis